### PR TITLE
maven jar plugin with default manifest

### PR DESCRIPTION
**Pull request by marmoure**
using the default manifest 
```xml
<plugin>
    <artifactId>maven-jar-plugin</artifactId>
    <version>2.6</version>
    <configuration>
        <useDefaultManifestFile>true</useDefaultManifestFile>
    </configuration>
</plugin>
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -92,6 +93,29 @@
 
     <build>
         <plugins>
+            <plugin>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>2.6</version>
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <addClasspath>true</addClasspath>
```
Comment (adamretter) [inline pom.xml:102]: We don't want to add the classpath for this project. You only add the classpath to projects that have a `main` function (i.e. are executable), this project is a library.

File: pom.xml [hunk 2]
```diff
@@ -31,6 +31,7 @@
     <groupId>com.evolvedbinary.lucene.analyzer</groupId>
     <artifactId>OhAnalyzer</artifactId>
     <version>1.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
```
Comment (adamretter) [inline pom.xml:34]: Jar is the default - so you can exclude this line.

**Review by adamretter — CHANGES_REQUESTED** [review]
Some small improvements please

File: pom.xml [hunk 3]
```diff
@@ -92,6 +93,29 @@
 
     <build>
         <plugins>
+            <plugin>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>2.6</version>
```
[the remark this answers is not in the document]
Comment (marmoure) [reply pom.xml]: the tutorial i followed was from few years ago i didnt notice that

File: pom.xml [hunk 4]
```diff
@@ -92,6 +92,28 @@
...
+                            <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+                            <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+                        </manifest>
+                        <manifestEntries>
+                            <Build-Tag>${build-tag}</Build-Tag>
+                            <Git-Commit>${build-commit}</Git-Commit>
+                            <Git-Commit-Abbrev>${build-commit-abbrev}</Git-Commit-Abbrev>
+                            <Build-Version>${build-version}</Build-Version>
+                            <Build-Timestamp>${build-tstamp}</Build-Timestamp>
+                            <Source-Repository>${project.scm.connection}</Source-Repository>
```
Comment (adamretter) [inline pom.xml:110]: @marmoure The `project` properties you are referencing here need to be set. Take a look here:

1. You need an `<scm>` section. Example here - https://github.com/digital-preservation/utf8-validator/blob/master/pom.xml#L33
2. You need a `<description>`. Example here - https://github.com/digital-preservation/utf8-validator/blob/master/pom.xml#L16
3. You need a `<url>`. Example here -https://github.com/digital-preservation/utf8-validator/blob/master/pom.xml#L17

File: pom.xml [hunk 5]
```diff
@@ -23,154 +23,176 @@
...
+	<organization>
+		<name>Evolved Binary</name>
+	</organization>
+	<properties>
+		<maven.compiler.source>8</maven.compiler.source>
+		<maven.compiler.target>8</maven.compiler.target>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<lucene.version>4.10.4</lucene.version>
+		<git.account>evolvedbinary</git.account>
+		<git.base>github.com/${git.account}/${project.artifactId}</git.base>
```
Comment (adamretter) [inline pom.xml:46]: As per the eXist-db community call - I think we both agree that using properties for the SCM section makes it hard to understand as a human - can you just repeat the full and correct urls in the scm section and remove these `git.` properties please?

**Pull request merged**